addons/mail/static/src/discuss/call/common/call.scss

```diff
@@ -10,11 +10,22 @@
     &.o-minimized {
         height: 20%; // ensures that the view returns to the right height when resized
         min-height: #{"max(20%, 130px)"};
-        &.o-compact {
-            height: 10%;
-            min-height: #{"max(10%, 100px)"};
-        }
     }
+
+    &.o-compact {
+        height: 10%;
+        min-height: #{"max(10%, 100px)"};
+    }
+
+    &.o-collapsed {
+        height: fit-content;
+        min-height: fit-content;
+        max-height: max-content;
+    }
+}
+
+.o-discuss-Call-collapsibleBtn {
+    background-color: rgba(0, 0, 0, 0.5);
 }
```

Comment:
Instead use bootstrap class in xml. We avoid to write CSS rules as much as possible to reduce the size of the bundle files.

Reply:
Bootstrap doesn't have a utility class that provides the specific background color I need with `rgba(0, 0, 0, 0.5)`. While `.bg-dark` exists, it uses a different color code. So using a custom CSS class seems like a better approach in this case.

Reply:
You could try this combination `bg-black bg-opacity-50`

Reply:
Already tried that combination of `bg-black` and `bg-opacity-50` but bg-dark has different color code than `#000` and it's more of dark blue than black, so that combination won't work here